kubernetes/jasperreports-server-service-deployment.yaml

```diff
@@ -51,96 +52,94 @@ spec:
       namespace: jaspersoft
     spec:
       containers:
-      - envFrom:
-        - configMapRef:
-            name: jasper-config
-        image: jasperserver-pro:7.8.0
-        name: jasperserver-pro
-        imagePullPolicy: IfNotPresent
-        ports:
-        - containerPort: 8080
-          name: http
-          protocol: TCP
-        - containerPort: 8443
-          name: https
-          protocol: TCP
-        resources:
-          limits:
-            #cpu: 200m
-            memory: 6Gi
-          requests:
-            #cpu: 100m
-            memory: 2Gi
-        volumeMounts:
-        - mountPath: /usr/local/share/jasperserver-pro/license
-          name: license
-          readOnly: true
-        - mountPath: /usr/local/share/jasperserver-pro/keystore
-          name: jasperserver-pro-keystore-files-secret
-          readOnly: true
-# Add if additional default_master.propeties in a Secret
-#        - mountPath: /usr/local/share/jasperserver-pro/deploy-customization
-#          name: jasperserver-pro-additional-master-secret
-#          readOnly: true
-        - mountPath: /usr/local/share/jasperserver-pro
-          name: jasperserver-pro-volume
-          readOnly: true
-        livenessProbe:
-          httpGet:
-            path: /jasperserver-pro/login.html
-            port: 8080
-          failureThreshold: 20
-          periodSeconds: 10
-          initialDelaySeconds: 60
-        readinessProbe:
-            httpGet:
-              path: /jasperserver-pro/login.html
-              port: 8080
+        - envFrom:
+            - configMapRef:
+                name: jasper-config
+          image: jasperserver-pro:7.8.0
+          name: jasperserver-pro
+          imagePullPolicy: IfNotPresent
+          ports:
+            - containerPort: 8080
```

Comment:
Should these still be 8080/8443 if we made changes commented on above?

Reply:
Yes , container port is still 8080 only